tests/test_pipenv.py

```diff
@@ -963,8 +1003,10 @@ def test_complex_deps_lock_and_install_properly(self, pypi):
     @pytest.mark.lock
     @pytest.mark.requirements
     @pytest.mark.complex
+    @pytest.mark.skipif(not WE_HAVE_INTERNET, reason='does not work without Internet')
     def test_complex_lock_deep_extras(self, pypi):
         # records[pandas] requires tablib[pandas] which requires pandas.
```

Comment:
Do we need to refrain from passing mocked pypi here?

Reply:
Yes